DESCRIPTION

```diff
@@ -10,7 +10,19 @@ Description: Implements an individual-based model of COVID-19 transmission.
 License: MIT + file LICENSE
 Encoding: UTF-8
 LazyData: true
+Remotes:  
+    mrc-ide/individual@0.0.8,
+    mrc-ide/squire@v0.5.3
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.1.1
 Suggests:
-    testthat
+    testthat,
+    mockery
+Imports:
+    dplyr,    
+    ggplot2,   
+    tibble,   
+    tidyr,
+    individual,
+    reshape2,
+    squire
```

Comment:
I think we can remove a bunch of these dependencies for the early versions? It should make building a lot faster

```suggestion
    individual,
    squire
```